pipeline/experiment.py

```diff
@@ -176,61 +176,37 @@ class SessionComment(dj.Manual):
 
 
 # ---- behavioral trials ----
-#   (perhaps not relevant)
 
 @schema
-class TrialInstruction(dj.Lookup):
-    definition = """
-    # Instruction to mouse 
-    trial_instruction  : varchar(8) 
-    """
-    contents = zip(('left', 'right', 'none'))
-
-
-@schema
-class Choice(dj.Lookup):
+class BehaviorTrial(dj.Imported):
     definition = """
-    # Choice of the mouse 
-    choice  : varchar(8) 
+    -> SessionTrial
+    ----
+    -> TaskProtocol
     """
-    contents = zip(('left', 'right', 'middle', 'none'))
 
 
 @schema
-class Outcome(dj.Lookup):
+class TrialOutcome(dj.Imported):
     definition = """
-    outcome : varchar(32)
+    -> BehaviorTrial
+    ---
+    outcome: enum('correct', 'incorrect')
     """
-    contents = zip(('hit', 'miss', 'ignore'))
 
 
 @schema
-class EarlyLick(dj.Lookup):
+class TrialObject(dj.Imported):
     definition = """
-    early_lick  :  varchar(32)
+    -> BehaviorTrial
+    -> lab.ExperimentObject
     ---
-    early_lick_description : varchar(4000)
+    distance_x: decimal(8,4)  # (mm) distance of object from the animal's head (todo: use proper lab's terminology here)
+    distance_y: decimal(8,4)  # (mm) distance of object from the animal's head (todo: use proper lab's terminology here)
```

Comment:
I'm assuming there is one object associated to the session (e.g., wall), described in `ExperimentObject` and all the trial parameters are defined here. So if a panel object has two different potential textures, the texture parameters is defined here. 
```suggestion
# Distances are from the animal's face (0 is centered on whisker pad) to the object's center / tip.
    distance_x: decimal(8,4)  # (mm) 
    distance_y: decimal(8,4)  # (mm) 
    distance_z: decimal(8,4)  # (mm) 
    azimuth: decimal(8,4)  # (degrees) 
    direction= null: varchar(50) # (forward, backward, looming ...) - If not static
    texture=null: varchar(50)  # (rough / smooth / grit density) - For texture panels
    strength=null: decimal(8,4)  # (in g) - for von Freys
    temperature=null: decimal(8,4)  # (degrees Celsius) - for heat/cold experiments  
```

Reply:
move `azimuth: decimal(8,4)  # (degrees) ` property to `ExperimentObject` as discussed today.

Reply:
Looks good